documentation/ztp-for-factories/ztp-for-factory-development.adoc

```diff
@@ -62,7 +62,7 @@ You can customize the parameter `git-revision=<BRANCH>` to point to your own bra
 
 ```sh
 export KUBECONFIG=/root/.kcli/clusters/test-ci/auth/kubeconfig
-tkn pipeline start -n spoke-deployer -p git-revision=main -p spokes-config="$(cat /root/amorgant/ztp-pipeline-relocatable/hack/deploy-hub-local/spokes.yaml)" -p kubeconfig=${KUBECONFIG} -w name=ztp,claimName=ztp-pvc --timeout 5h --use-param-defaults deploy-ztp-hub
+tkn pipeline start -n spoke-deployer -p spokes-config="$(cat /root/amorgant/ztp-pipeline-relocatable/hack/deploy-hub-local/spokes.yaml)" -p kubeconfig=${KUBECONFIG} -w name=ztp,claimName=ztp-pvc --timeout 5h --use-param-defaults deploy-ztp-hub
```

Comment:
This should probably have: `-p ztp-container-image="quay.io/ztpfw/pipeline:main"`

Although that's the default

Reply:
updated.
The example helps.